src/config.ts

```diff
@@ -31,14 +32,37 @@ let config: InternalConfig = {
 
   // called when getBy* queries fail. (message, container) => Error
   getElementError(message, container) {
+    const playgroundUrl = (() => {
+      try {
+        return (
+          message &&
```

Comment:
Why does the value of 'message' matter here?

Reply:
It's because of the usage of `getElementError` in `makeSingleQuery`:

```javascript
      const elementStrings = els
        .map(element => getElementError(null, element).message)
        .join('\n\n')
```

You can see that it's running through multiple element strings and joining the output. If we don't have the `message &&` part it would print the playground URL multiple times, for example:

```
Open this markup in the Testing-Library Playground:
https://testing-playground.com/#markup=JIOwxgTgpgtlIBcAEA3AhgGwK5SA

<button>
   Decrement value
</button>

Open this markup in the Testing-Library Playground:
https://testing-playground.com/#markup=CIUwxgTiC2IHYBcAEA3AhgGwK4iA

<button>
   Reset value
</button>

Open this markup in the Testing-Library Playground:
https://testing-playground.com/#markup=EoUwziAuAEBuCGAbAriIA
```

and this seams like a bit too much :-)